vkcs/compute/resource_instance_test.go

```diff
@@ -760,6 +760,51 @@ func TestResourceInstanceBlockDevicesV2(t *testing.T) {
 	}
 }
 
+func TestResourceComputeInstanceCloudMonitoring(t *testing.T) {
+	var instance servers.Server
+	resourceName := "vkcs_compute_instance.instance_1"
+	monitoringResourceName := "vkcs_cloud_monitoring.basic"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV6ProviderFactories: acctest.AccTestProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckComputeInstanceDestroyV6,
+		Steps: []resource.TestStep{
+			{
+				Config: acctest.AccTestRenderConfig(testAccComputeInstanceCloudMonitoring, map[string]string{"Metadata": testMetadata}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeInstanceExistsV6(resourceName, &instance),
+					testAccCheckComputeInstanceMetadata(&instance, "key1", "value1"),
```

Comment:
It would be better to check monitoring attributes here, as it's an acceptance test it should be clear which feature is being tested, so, all these metadata checks look a bit like overhead and better to test it within some metadata test if it's needed.

Reply:
Split into thow tests. In basic check only service_user_id is exists in instance metadata (front requires it)